src/Entity/ListBuilder/EdgeEntityListBuilder.php

```diff
@@ -185,4 +191,14 @@ protected function getDisplaySettings(): array {
       ->getRawData();
   }
 
+  /**
+   * Returns the cache max age.
+   *
+   * @return int
+   *   The cache max age.
+   */
+  protected function getCacheMaxAge() {
+    return 0;
```

Comment:
How about providing a sane default?

```php
<?php
    $config = $this->configFactory->get("apigee_edge.{$this->entityTypeId}_app_settings");
    if (!$config->isNew()) {
      return  $config ->get('cache_expiration');
    }
   return 0;
```

Reply:
I see but `cache_expiration` is not available for all entity types. It's up to the Edge entity list to implement the config for this and the list builder. So `0` makes sense here since we don't want to cache by default.

Reply:
> I see but cache_expiration is not available for all entity types
let me challenge that... if this is true, why do we have a 'getCacheMaxAge()' in the base class if it not leverage by 100% of the sub-classes?